### PR TITLE
Add Firefox 77.0

### PR DESCRIPTION
**Pull request by SoftCreatR**
Auto Generated by softcreatr@winget.it on Tue, 02 Jun 2020 12:35:29 +0000

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1471)

### CODE REVIEW

**Comment by superusercode** [comment]
Why not use the MSI installer or the full fat exe for deployment that is an inno installer? 

Please see https://github.com/microsoft/winget-pkgs/issues/333

File: manifests/Mozilla/Firefox/77.0.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Version: 77.0
+
+InstallerType: exe
+Installers:
+  - Arch: x64
+    Url: https://download-installer.cdn.mozilla.net/pub/firefox/releases/77.0/win64/en-US/Firefox%20Setup%2077.0.exe
+    Sha256: 859EFFFC777B2260250C982036920BAEF79369CBCE08822BA8700DB1EC7E8B88
+    Switches: 
+      Silent: "-ms -ma"
+      SilentWithProgress: "-ms -ma"
```
Comment (superusercode) [inline manifests/Mozilla/Firefox/77.0.yaml:23]: inno install type works here

**Comment by EwoutH** [comment]
Please also add the Arm64 version:
```
  - Arch: arm64
    Url: https://download-installer.cdn.mozilla.net/pub/firefox/releases/77.0/win64-aarch64/en-US/Firefox%20Setup%2077.0.exe
    Sha256: 817C0597146135B3E4D8A94DA4F85F1221CF04FA5D9DBF6F27FDB66FCF43A524
```

**Comment by SoftCreatR** [comment]
@ewouth There's currently no support for more than 1 installer.

**Pull request closed without merging**